src/core/public/doc_links/doc_links_service.ts

```diff
@@ -46,10 +46,84 @@ export class DocLinksService {
     const OPENSEARCH_DOCS = `https://opensearch.org/docs/opensearch/`;
     const OPENSEARCH_DASHBOARDS_DOCS = `https://opensearch.org/docs/dashboards/`;
```

Comment:
We shouldn't need the non-versioned links anymore, right?

Reply:
I am plan to have another PR to clean up all unnecessary URLs. I will address this in next revision